packages/schematics/angular/e2e/files/protractor.conf.js.template

```diff
@@ -17,7 +17,7 @@ exports.config = {
   },
   directConnect: true,
   SELENIUM_PROMISE_MANAGER: false,
-  baseUrl: 'http://localhost:4200/',
+  baseUrl: 'http://localhost:0/',
```

Comment:
I highly doubt this is valid.

Probably this works because the baseUrl is set in the protoactor builder. And thus this line can probably be dropped. 

Same as the others.

Reply:
I think you can leave it as it was since in this case port 0 in the baseUrl is technically invalid

Reply:
@alan-agius4 are you sure this is invalid? See https://github.com/angular/angular-cli/commit/6db7038c1bb07604348fbe804534561c0a6f7b7a

I'm looking at this again because I'm getting port-in-use errors again when running e2e tests concurrently...

Reply:
(although I should only be changing it in the e2e tests, not in this schematics file)